ResearchKit/ActiveTasks/ORKHealthQuantityTypeRecorder.m

```diff
@@ -52,20 +52,6 @@ @interface ORKHealthQuantityTypeRecorder () {
 
 @end
 
-#ifdef __IPHONE_10_0
```

Comment:
This was only needed during last summer so folks could develop with either Xcode 8 beta or Xcode 7. I've removed it in a branch that is waiting for acceptance of my other pull requests.